geonode_dominode/geonode_dominode/templates/groups/group_detail_override.html

```diff
@@ -0,0 +1,109 @@
+{% extends "groups/group_detail.html" %}
+{% load i18n %}
+
+{% block body_outer %}
+<div class="page-header">
+  <h2 class="page-title">{{ object.title|default:object.slug }}</h2>
+  <p>
+    {% trans "Last Modified" %}: {{ object.last_modified }}
+  </p>
+</div>
+<div class="row">
+  <div class="col-md-8 group-details">
+    <div class="row">
+      <div class="col-xs-4 col-md-3 group-logo">
+      {% if object.logo_url %}
+        <img src="{{ object.logo_url }}" alt="{{ object.title }}" />
+      {% else %}
+        <!-- <p>{% trans "This group has not created a logo." %}</p> -->
+      {% endif %}
+      </div>
+      <div class="col-xs-8 col-md-9 group-details">
+        <p class="group-desc">{{ object.description }}</p>
+        {% if object.keyword_list %}
+        <p>
+          {% for keyword in object.keyword_list %}
+            <span class="label label-default">{{ keyword }}</span>
+          {% endfor %}
+        </p>
+        {% else %}
+        {% endif %}
+        {% if object.email %}
+        <p><i class="fa fa-envelope-o"></i>
+          <a href="mailto:{{ object.email }}">{{ object.email }}</a>
+        </p>
+        {% endif %}
+
+        {% if object.categories.exists %}
+        <p>
+          {% for category in object.categories.all %}
+            <a href="{{ category.get_absolute_url }}"><span class="label label-info">{{ category.name }}</span></a>
+          {% endfor %}
+        </p>
+        {% endif %}
+
+      </div>
+    </div>
+  </div>
+  <div class="col-md-4">
+    <ul class="list-group">
+      {% if is_manager %}
+      <li class="list-group-item"><a href="{% url "group_update" object.slug %}">{% trans "Edit Group Details" %}</a></li>
+      <li class="list-group-item"><a href="{% url "group_members" object.slug %}">{% trans "Manage Group Members" %}</a></li>
+      <li class="list-group-item"><a href="{% url "group_remove" object.slug %}">{% trans "Delete this Group" %}</a></li>
+      {% endif %}
+      {% if can_view %}
+      <li class="list-group-item"><a href="{% url "group_activity" object.slug %}">{% trans "Group Activities" %}</a></li>
+      {% endif %}
+      {% if perms.groups.can_sync_geoserver %}
+      <li class="list-group-item">
+        <form action="{% url "cli_sync_geoserver" %}" method=post>
+          {% csrf_token %}
+          <input type="hidden" name="group-slug" value="{{ object.slug }}" />
+          <input type="hidden" name="redirect" value="{{ request.path }}" />
+          <input type="submit" class="btn btn-primary" data-toggle="modal" data-target="#sync-geoserver-modal" value="{% trans "Sync GeoServer" %}" />
```

Comment:
remove these data-attrs that deal with the modal

Reply:
Oh yeah, forgot about that.